divi/src/defaultValues.h

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2017-2020 The DIVI Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef _DEFAULTVALUES_H
+#define _DEFAULTVALUES_H
+
+#include <primitives/block.h>
+
```

Comment:
From the name of the file it doesn't seem like the block class actually belongs here at all. After all, it's just a bunch of constants related to various parts of the core

Reply:
done